README.md

```diff
@@ -54,6 +54,30 @@ server = Server.new
 server.emit :user_connected, { name: 'Somebody', ip: '127.0.0.1' }
 ```
 
+Config
+------
+
+### Limiting listeners
+
+You can set a limit to prevent adding listeners by using:
+```ruby
+class Server
+  include Emittr::Events
+
+  listeners_limit 20
```

Comment:
What do you think we change the name to `max_listeners`? It's the same name used by Node's implementation.